### PR TITLE
Produce reusable output from cperepo tool

### PR DESCRIPTION
**Pull request by andrewpollock**
Full overhaul to make this generally accessible and useful

### CODE REVIEW

File: vulnfeeds/cmd/cperepos/main.go [hunk 1]
```diff
@@ -121,31 +213,59 @@ func main() {
...
+	if err != nil {
+		Logger.Fatalf("Failed to load %s: %v", *CPEDictionaryFile, err)
 	}
-	sort.SliceStable(descriptions, func(i, j int) bool {
-		return DescriptionFrequency[descriptions[i]] > DescriptionFrequency[descriptions[j]]
-	})
-	for _, d := range descriptions {
-		fmt.Printf("%s,%d\n", d, DescriptionFrequency[d])
+
+	ProductToRepo, DescriptionFrequency, ProductToVendor := analyzeCPEDictionary(CPEDictionary)
```
Comment (oliverchang) [inline vulnfeeds/cmd/cperepos/main.go:241]: nit: these should be `camelCase` names

File: vulnfeeds/cmd/cperepos/main.go [hunk 2]
```diff
@@ -121,31 +213,59 @@ func main() {
...
+
+	ptrmf, err := os.Create(filepath.Join(*OutputDir, "cpe_product_to_repo.json"))
```
Comment (oliverchang) [inline vulnfeeds/cmd/cperepos/main.go:243]: both ptrmf and crdff are a little hard to read and parse as a human :) Consider just using a little more descriptive names. e.g. `mapWriter`, `freqWriter`.

**Review by oliverchang — APPROVED** [review]

File: vulnfeeds/cmd/cperepos/main.go (hunk 1 above)
Comment (andrewpollock) [reply vulnfeeds/cmd/cperepos/main.go]: Addressed in [ab29b63](https://github.com/google/osv.dev/pull/853/commits/ab29b63a7ae6d672086e812604f58170059140b1)

File: vulnfeeds/cmd/cperepos/main.go (hunk 2 above)
Comment (andrewpollock) [reply vulnfeeds/cmd/cperepos/main.go]: Addressed in [ab29b63](https://github.com/google/osv.dev/pull/853/commits/ab29b63a7ae6d672086e812604f58170059140b1)

**Pull request merged**